### ISSUE TITLE
Fix qeq_fire fails to compile with Intel (16) 

### ISSUE THREAD

**Issue by crtrott** [lammps/lammps#42]
I got following failure to compile:
```
mpicxx -g -O3 -restrict   -DLAMMPS_GZIP -DLMP_USER_OMP -DLMP_PYTHON -DLMP_MPIIO -DLMP_KOKKOS  -DMPICH_SKIP_MPICXX -DOMPI_SKIP_MPICXX=1    -I/projects/install/rhel6-x86_64/sems/compiler/python/2.7.9/include/python2.7 -I/projects/install/rhel6-x86_64/sems/compiler/python/2.7.9/include/python2.7 -I./ -I../../lib/kokkos/core/src -I../../lib/kokkos/containers/src -I../../lib/kokkos/algorithms/src  --std=c++11   -c ../fix_qeq_fire.cpp
../fix_qeq_fire.cpp(228): warning #3494: a user-provided literal suffix must begin with "_"
        sprintf(str,"Charges did not converge at step "BIGINT_FORMAT
                    ^

../fix_qeq_fire.cpp(228): error: user-defined literal operator not found
        sprintf(str,"Charges did not converge at step "BIGINT_FORMAT
                    ^

compilation aborted for ../fix_qeq_fire.cpp (code 2)
```

This is with the following modules loaded (which have the compiler versions and so in the name):
```  
  1) gcc/4.7.2/base                3) intel/16.0.1/openmpi/1.10.1   5) gdb/7.9.1
  2) intel/16.0.1/base             4) git/2.1.3                     6) python/2.7.9
```

This is the list of enabled packages:
```
Installed YES: package ASPHERE
Installed YES: package BODY
Installed YES: package CLASS2
Installed YES: package COLLOID
Installed YES: package COMPRESS
Installed YES: package CORESHELL
Installed YES: package DIPOLE
Installed  NO: package GPU
Installed YES: package GRANULAR
Installed  NO: package KIM
Installed YES: package KOKKOS
Installed YES: package KSPACE
Installed YES: package MANYBODY
Installed YES: package MC
Installed  NO: package MEAM
Installed YES: package MISC
Installed YES: package MOLECULE
Installed YES: package MPIIO
Installed YES: package OPT
Installed YES: package PERI
Installed  NO: package POEMS
Installed YES: package PYTHON
Installed YES: package QEQ
Installed  NO: package REAX
Installed YES: package REPLICA
Installed YES: package RIGID
Installed YES: package SHOCK
Installed YES: package SNAP
Installed YES: package SRD
Installed  NO: package VORONOI
Installed YES: package XTC

Installed  NO: package USER-ATC
Installed  NO: package USER-AWPMD
Installed YES: package USER-CG-CMM
Installed  NO: package USER-COLVARS
Installed  NO: package USER-CUDA
Installed YES: package USER-DIFFRACTION
Installed YES: package USER-DPD
Installed YES: package USER-DRUDE
Installed YES: package USER-EFF
Installed YES: package USER-FEP
Installed  NO: package USER-H5MD
Installed  NO: package USER-INTEL
Installed YES: package USER-LB
Installed YES: package USER-MGPT
Installed YES: package USER-MISC
Installed YES: package USER-MOLFILE
Installed YES: package USER-OMP
Installed YES: package USER-PHONON
Installed  NO: package USER-QMMM
Installed YES: package USER-QTB
Installed  NO: package USER-QUIP
Installed YES: package USER-REAXC
Installed  NO: package USER-SMD
Installed YES: package USER-SMTBQ
Installed YES: package USER-SPH
Installed YES: package USER-TALLY
Installed  NO: package USER-VTK
```

If you need more info let me know.

**Comment by akohlmey** [comment]
seen that one before and fixed it in different QEQ variants. the fix is trivial:
```
diff --git a/src/QEQ/fix_qeq_fire.cpp b/src/QEQ/fix_qeq_fire.cpp
index af9ca65..46440fc 100644
--- a/src/QEQ/fix_qeq_fire.cpp
+++ b/src/QEQ/fix_qeq_fire.cpp
@@ -225,7 +225,7 @@ void FixQEqFire::pre_force(int vflag)
   if (comm->me == 0) {
     if (iloop == maxiter) {
       char str[128];
-      sprintf(str,"Charges did not converge at step "BIGINT_FORMAT
+      sprintf(str,"Charges did not converge at step " BIGINT_FORMAT
                  ": %lg",update->ntimestep,enegchk);
       error->warning(FLERR,str);
     }
```

**Comment by akohlmey** [comment]
fixed in commit 0ea702cb6a59ceb7dc3151a7bf6200969ba2c7e4
